contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
   secretName: my-secret
+  configMapName: my-configmap
+  serviceName: my-service
+  osbGuid: b58d64d2-8e01-42b5-a287-bc9ee1860f9d
```

Comment:
I'm +1 on generating these for the user. I will open a separate issue for this. As long as this is an accurate example though, I'd leave it in here for now. It may be ugly, but it's reality.